cypress/e2e/invoices/approve-invoice.cy.js

```diff
@@ -66,12 +66,12 @@ describe('ui-invoices: Approve invoice', () => {
   it('C10945 Approve invoice (thunderjet)', { tags: [testType.criticalPath, devTeams.thunderjet] }, () => {
     const transactionFactory = new Transaction();
     Invoices.searchByNumber(invoice.invoiceNumber);
-    Helper.selectFromResultsList();
+    Invoices.selectInvoice(invoice.invoiceNumber);
     Invoices.approveInvoice();
     // check transactions after approve
     cy.visit(TopMenu.fundPath);
     Helper.searchByName(defaultFund.name);
-    Helper.selectFromResultsList();
+    Funds.selectFund(defaultFund.name);
```

Comment:
![image](https://user-images.githubusercontent.com/99860864/208080425-0ecbe96d-eb6c-4cb8-8c1d-215dfd987428.png)
you have eslint problems